### DISCUSSION TITLE
Discord server for Software Mansion open source projects

### DISCUSSION THREAD

**Discussion by andreialecu** [software-mansion/react-native-reanimated#2363] [General]
Hey guys,

I wonder if you ever considered starting a Discord server. It would make communicating with the community easier, especially for troubleshooting new releases.

Callstack has one and is pretty useful: https://reactnative.dev/help#company-based-communities

**Comment by MilanBarande** [comment]
If like me you reach this issue from google: https://discord.com/invite/2gjSqPQc9Q

**Reply by MatiPl01** [reply]
Yup, this is the correct invitation link that you can use to join the server. Thanks @MilanBarande for posting it here!
